extensions/wikia/Recirculation/js/recirculation.js

```diff
@@ -84,10 +87,27 @@ require([
 	});
 
 	// fetch data for all recirculation modules
-	// TODO lazy load some data on scroll
 	liftigniter.fetch('ns');
-	liftigniter.fetch('wiki');
-	discussions.fetch();
+
+	window.addEventListener('scroll', function lazyLoadHandler() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Make this function anonymous by removing its name: 'function() {...}'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3ANamedFunctionExpression)